multiqc/modules/dragen/coverage_metrics.py

```diff
@@ -0,0 +1,240 @@
+import re
+from collections import OrderedDict, defaultdict
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import linegraph, bargraph, table
+
+# Initialise the logger
+import logging
+log = logging.getLogger(__name__)
+
+from .utils import make_headers, Metric
+
+
+NAMESPACE = 'DRAGEN coverage'
+
+
+class DragenCoverageMetrics(BaseMultiqcModule):
+    def add_coverage_metrics(self):
+        data_by_phenotype_by_sample = defaultdict(dict)
+
+        for f in self.find_log_files('dragen/wgs_coverage_metrics'):
+            data_by_phenotype = parse_wgs_coverage_metrics(f)
+            if f['s_name'] in data_by_phenotype:
+                log.debug('Duplicate sample name found! Overwriting: {}'.format(f['s_name']))
+            self.add_data_source(f, section='stats')
+            data_by_phenotype_by_sample[f['s_name']].update(data_by_phenotype)
+
+        # Filter to strip out ignored sample names:
+        data_by_phenotype_by_sample = self.ignore_samples(data_by_phenotype_by_sample)
+
+        # Merge tumor and normal data:
+        data_by_sample = defaultdict(dict)
+        for sn in data_by_phenotype_by_sample:
+            for phenotype in data_by_phenotype_by_sample[sn]:
+                new_sn = sn
+                if phenotype == 'normal':
+                    new_sn = sn + '_normal'
+                data_by_sample[new_sn] = data_by_phenotype_by_sample[sn][phenotype]
+
+        if not data_by_sample:
+            return
+        log.info('Found DRAGEN coverage metrics for {} DRAGEN output prefixes'.format(len(data_by_sample)))
+
+        all_metric_names = set()
+        for sn, sdata in data_by_sample.items():
+            for m in sdata.keys():
+                all_metric_names.add(m)
+        gen_stats_headers, own_tabl_headers = make_headers(all_metric_names, COV_METRICS)
+
+        self.general_stats_addcols(data_by_sample, gen_stats_headers, namespace=NAMESPACE)
+
+        self.add_section(
+            name='Coverage metrics',
+            anchor='dragen-cov-metrics',
+            description='Coverage metrics over a region (where the region can be a target region, '
+                        'a QC coverage region, or the whole genome). Press the `Help` button for details.',
+            helptext="""The following criteria are used when calculating coverage:
+            - Duplicate reads and clipped bases are ignored.
+            - Only reads with MAPQ > min MAPQ and bases with BQ > min BQ are considered
+            Considering only bases usable for variant calling, i.e. excluding:
+                        1. clipped bases,
+                        2. bases in duplicate reads,
+                        3. reads with MAPQ < min MAPQ (default 20),
+                        4. bases with BQ < min BQ (default 10) 
+                        5. reads with MAPQ = 0 (multimappers); 
+                        6. overlapping mates are double-counted.
+                        """.replace('\n', '<br>'),
```

Comment:
Should be able to use use regular markdown (maybe needs a little more whitespace?):

```suggestion
            helptext="""
            The following criteria are used when calculating coverage:
            
            * Duplicate reads and clipped bases are ignored.
            * Only reads with `MAPQ` > `min MAPQ` and bases with `BQ` > `min BQ` are considered
            
            Considering only bases usable for variant calling, _i.e._ excluding:
            
            1. Clipped bases
            2. Bases in duplicate reads
            3. Reads with `MAPQ` < `min MAPQ` (default `20`)
            4. Bases with `BQ` < `min BQ` (default `10`)
            5. Reads with `MAPQ` = `0` (multimappers)
            6. Overlapping mates are double-counted
            
            """,
```